x/axelarcork/keeper/keeper.go

```diff
@@ -150,7 +151,7 @@ func (k Keeper) GetScheduledAxelarCorks(ctx sdk.Context, chainID uint64) []*type
 			Validator:   val.String(),
 			Cork:        &cork,
 			BlockHeight: blockHeight,
-			Id:          id,
+			Id:          hex.EncodeToString(id),
```

Comment:
Looking at this commit, we've updated the output encoding, but are there any queries that rely on the ID as input which were expecting bytes before and now need to convert from a hex string into bytes?